open-api/src/testFixtures/java/org/apache/iceberg/rest/RESTCatalogServer.java

```diff
@@ -119,6 +121,11 @@ public void start(boolean join) throws Exception {
         new Server(
             PropertyUtil.propertyAsInt(catalogContext.configuration, REST_PORT, REST_PORT_DEFAULT));
     httpServer.setHandler(context);
+    Arrays.stream(httpServer.getConnectors())
+        .forEach(
+            connector -> {
+              ((ServerConnector) connector).setReusePort(true);
+            });
```

Comment:
nit: Normal for loop is simpler in this case. Feel free to ignore:
```java
    for (Connector connector : httpServer.getConnectors()) {
      ((ServerConnector) connector).setReusePort(true);
    }
```

Reply:
@jackylee-ch I will merge the PR after the above nit suggestion is applied.

Reply:
Done, thanks.